.github/workflows/bump_version.yml

```diff
@@ -29,79 +29,97 @@ jobs:
 
       - name: Fetch all tags and branches
         run: |
-          git fetch --tags
+          git fetch --unshallow --tags
           git fetch origin '+refs/heads/*:refs/remotes/origin/*'
-    
-      - name: Download latest auto
+
+      - name: Use Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20
+
+      # - name: Download latest auto
+      #   run: |
+      #     auto_download_url="$(curl -fsSL https://api.github.com/repos/intuit/auto/releases/latest | jq -r '.assets[] | select(.name == "auto-linux.gz") | .browser_download_url')"
+      #     wget -O- "$auto_download_url" | gunzip > ~/auto
+      #     chmod a+x ~/auto
+
+      - name: Install auto
         run: |
-          auto_download_url="$(curl -fsSL https://api.github.com/repos/intuit/auto/releases/latest | jq -r '.assets[] | select(.name == "auto-linux.gz") | .browser_download_url')"
-          wget -O- "$auto_download_url" | gunzip > ~/auto
-          chmod a+x ~/auto
+          npm install auto
 
-      - name: Get latest release version
-        id: latest-release
+      - name: Create Release
```

Comment:
That's not what we're doing here though, right? We're computing the version / testrun